### ISSUE TITLE
P100, P110 overheat state in HA automation

### ISSUE THREAD

**Issue by mihing** [petretiandrea/home-assistant-tapo-p100#380]
## Version of the integration
1.6.0

## Configuration
p100, p110

```yaml
condition:
  - type: is_not_hot
    condition: device
    device_id: xxxx
    entity_id: binary_sensor.tp_socket_04_overheat
    domain: binary_sensor

```

## Describe the bug

HA automation expects a state of "off", but integration provides a value of "False".
Sorry for poor english. 

## Debug log

```text

type: is_not_hot
condition: device
device_id: xxxx
entity_id: binary_sensor.tp_socket_04_overheat
domain: binary_sensor

entity_id/0
Executed: 11:44:46
Result:

result: false
state: 'False'
wanted_state: 'off'

```

**Comment by petretiandrea** [comment]
As reported here https://www.home-assistant.io/integrations/binary_sensor/:
`For example, a binary sensor for a window may report a value of open or closed, a switch on or off, a condition true or false.`

So I think true/false and false is a valid value, right?

**Comment by Jens987** [comment]
Not really - tried to change state values to on or off using developer tools, and then it looks right and behaves the right way in lovelace as sensor

**Comment by codyc1515** [comment]
Let me check on this.

**Comment by petretiandrea** [comment]
Fixed on 2.9.0
